docs/website/documentation/faq.md

```diff
@@ -20,3 +20,29 @@ Supplying the answers are a TODO.
 ## How do I supply answers?
 
 Click the "Edit this Page" link in the bottom right.
+
+
+# Common Problems:
+
+## java.lang.OutOfMemoryError: unable to create new native thread
+
+You could encounter this error when running with many entities.
+
+Please **increase the ulimit** if you see such error:
+
+On the VM running Apache Brooklyn, we recommend ensuring nproc and nofile are reasonably high (e.g. higher than 1024, which is often the default).
+
+Run `ulimit -a` to see the current limits.
+
```

Comment:
The correct setup for limits.conf is distribution specific. Maybe, it is better to point out only the recommended values without suggesting changing OS files and ask the user to consult distro's documentation.